code/modules/mob/language/german.dm

```diff
@@ -0,0 +1,393 @@
+/datum/language/german
+	name = LANGUAGE_NEO-GERMANIC
```

Comment:
This seems like a define, remove the "-".

Reply:
`LANGUAGE_GERMAN` so it conforms with the other languages.

Reply:
Thank you for the feedback